admission-webhook/deploy/gmsa-crd.yml

```diff
@@ -49,6 +92,9 @@ spec:
                     type: string
                   Sid:
                     type: string
+  conversion:
+    strategy: None
+  scope: Namespaced
```

Comment:
What's the difference between this scope and https://github.com/kubernetes-sigs/windows-gmsa/pull/44/files#diff-422342a87b72c2af7384f2cf5d6619f8958ff765dc83be7e774fa4ded59ef415R101

I believe these have been cluster scoped up until now?

Reply:
Oh yea, this should be cluster scoped